src/darwin/Framework/CHIP/gen/CHIPClientCallbacks.cpp

```diff
@@ -493,6 +493,9 @@ bool emberAfReadAttributesResponseCallback(ClusterId clusterId, uint8_t * messag
                             data[i].NodeId = emberAfGetInt64u(message, 0, messageLen);
                             message += 8;
                             CHECK_MESSAGE_LENGTH(8);
+                            data[i].Label = chip::ByteSpan(message, 32);
```

Comment:
Is the ByteSpan used after message receipt or is it a synchronous flow? Byte spans do not own data. I cannot see a spot where the data is actually copied into owned memory until the event is handled. Does this mean any usage of the label (for the lifetime of the underlying message) is capped to a synchronous call chain from the receipt of the message?